R/SSexecutivesummary.r

```diff
@@ -260,25 +265,41 @@ SSexecutivesummary <- function(replist,
       } else {
         es.a <- data.frame(years_minus_final, catch, total.catch, total.dead)
       }
-      colnames(es.a) <- c("Year", fleet.names, "Total Landings", "Total Dead")
+      colnames(es.a) <- c("Year", fleet.names, "Total Catch", "Total Dead")
 
       write.csv(es.a, file.path(csv.dir, csv_name), row.names = FALSE)
-      caption <- c(
-        caption,
-        "Recent landings by fleet, total landings summed across fleets, and the total mortality including discards."
-      )
+      if (is.null(add_text)) {
+        caption <- c(
+          caption,
+          "Recent landings by fleet, total landings summed across fleets, and the total mortality including discards."
```

Comment:
Couldn't we just change the default add_text to be `"model"` and have a single caption rather than the `if` `else` notation. So for instance, this caption would be "Recent landings by fleet, total landings summed across fleets, and the total mortality including discards included in the model" and users could pass instead `"northern California sub-area model"` and that would be used to replace model? We would basically just change every caption to have a paste at the at end rather than having if else statements for each caption.

Reply:
Commit [70be856](https://github.com/r4ss/r4ss/pull/836/commits/70be8568975e37fae6ac5246a0618932544827c0) simplifies the captions based on this suggestion